src/net.h

```diff
@@ -853,6 +856,26 @@ class CConnman
     uint64_t GetTotalBytesRecv() const;
     uint64_t GetTotalBytesSent() const EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
 
+    struct MsgStatsValue {
+        int msg_count;
+        uint64_t byte_count;
```

Comment:
May want to use std::atomic_uint64_t instead. 

Need to make a list of pros and cons (`atomic_uint64_t` vs `uint64_t`)